presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -338,107 +346,93 @@ public void testTimestampCreatedFromHive(StorageFormat storageFormat)
             throws Exception
     {
         String tableName = "test_timestamp_" + storageFormat.getName().toLowerCase(Locale.ENGLISH);
-        setupTimestampData(tableName, storageFormat);
+        createSimpleTimestampTable(tableName, storageFormat);
```

Comment:
The cmt msg says:

> - Rename two methods in TestHiveStorageFormats
>    - setupTimestampData to setupTimestampTable

I guess it would need an update